.ci/build_doc.bat

```diff
@@ -1,7 +0,0 @@
-set SPHINX_APIDOC_OPTIONS=inherited-members
```

Comment:
why are we deleting this? In the release 0.5.dev2 I updated this file and the changes are important to remove many docs warnings

Reply:
yes, thanks @PProfizi